frontend/.gitignore

```diff
@@ -4,7 +4,7 @@
 /node_modules
 /.pnp
 .pnp.js
-
+.env
```

Comment:
Why was this added?  For auto deployment, we need the environment to be picked up when the image is build _with no modification_ (since this will happen on github)

Reply:
I might have added it by mistake. Removing it right away.